src/font/face/freetype.zig

```diff
@@ -323,14 +322,6 @@ pub const Face = struct {
             // glyph properties before render so we don't render here.
             .render = !self.synthetic.bold,
 
-            // Disable bitmap strikes for now since it causes issues with
-            // our cell metrics and rasterization. In the future, this is
-            // all fixable so we can enable it.
-            //
-            // This must be enabled for color faces though because those are
-            // often colored bitmaps, which we support.
-            .no_bitmap = !self.face.hasColor(),
```

Comment:
Just noting I'm going to consider the implications of this a bit further. I don't remember the full history here.